internal/service/handler.go

```diff
@@ -2,21 +2,24 @@ package service
 
 import (
 	"github.com/kubev2v/migration-planner/internal/api/server"
+	"github.com/kubev2v/migration-planner/internal/config"
 	"github.com/kubev2v/migration-planner/internal/events"
 	"github.com/kubev2v/migration-planner/internal/store"
 )
 
 type ServiceHandler struct {
 	store       store.Store
 	eventWriter *events.EventProducer
+	cfg         *config.Config
 }
 
 // Make sure we conform to servers Service interface
 var _ server.Service = (*ServiceHandler)(nil)
 
-func NewServiceHandler(store store.Store, ew *events.EventProducer) *ServiceHandler {
+func NewServiceHandler(store store.Store, ew *events.EventProducer, cfg *config.Config) *ServiceHandler {
 	return &ServiceHandler{
 		store:       store,
 		eventWriter: ew,
+		cfg:         cfg,
```

Comment:
Why do you need cfg here?

Reply:
So I can take it from config.yaml in case env var is not set. This needs refactoring, I put TODO in code, I will do it in follow up PR.

Reply:
but we have a new service now. I don't see where this var is used in this service.

Reply:
https://github.com/kubev2v/migration-planner/pull/157/files#diff-662d8d27575cdd81927622a394d6c33a358b38f9de49c2dc594d5b7739131eb0R36

The hostname of the image service is defined either in `MIGRATION_PLANNER_URL` or in config.yaml

Reply:
Ah, only now I see it's wrong service, you are right, removing....